server/ldap_check.rb

```diff
@@ -14,14 +14,19 @@ class LdapCheck
 
   ## Setup the ldap connection object and save the members of the group.
 
-
   def initialize(args={})
 
     # This file will be provided in the build with values
     # suitable for the MCommunity groups.
     @default_conf_file = "./server/local/ldap.yml"
+    @default_cache_seconds = 300
```

Comment:
I understand that this value can be overridden by the conf. file, but how did you decide that the default number of seconds should be 300?